disco_aws_automation/disco_datapipeline.py

```diff
@@ -438,3 +446,21 @@ def _update_defaults(pipeline_objects, log_location=None, subnet_id=None):
         new_fields[DataPipelineConsts.SUBNET_ID_FIELD] = subnet_id
     if new_fields:
         add_default_object_fields(pipeline_objects, new_fields)
+
+
+def _update_daily_schedule(pipeline_objects, backup_period=None):
```

Comment:
Why is this called `_update_daily_schedule` if `backup_period` can be longer than a day